### DISCUSSION TITLE
Run time doubled even when re-running only 1 failed job

### DISCUSSION THREAD

**Discussion by wheeheee** [community/community#76304] [Actions]
### Select Topic Area

Bug

### Body

Choosing the option to re-run a single failed job does that, BUT when viewing the usage, the minutes spent on successful jobs are duplicated, exactly, adding up to nearly double in total run time. This seems unlikely to be the case if they were actually run because of jitter.

I was also viewing the summary page while it was running, and saw only the failed job being re-run. While I'm not a paying customer, this doesn't look nice for those who are, although I cannot confirm if this is reflected in the billable minutes.

It would be nice if only the re-runs were recorded in the usage page.

An example [here](https://github.com/wheeheee/MFCC.jl/actions/runs/6716402654/usage), and with more detail at actions/runner#2964.

**Comment by gtjoseph** [comment]
I just ran into this same issue.  Although our organization is in the free tier, the private repositories we use for handling security issues only get a limited number of free minutes.  A typical check job for us can be 10 hours long so having the usage double counted is a huge deal for us.
